pkg/golinters/exptostd/testdata/exptostd.go

```diff
@@ -0,0 +1,85 @@
+//golangcitest:args -Eexptostd
+package testdata
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"   // want `Import statement 'golang.org/x/exp/maps' can be replaced by 'maps'`
+	"golang.org/x/exp/slices" // want `Import statement 'golang.org/x/exp/slices' can be replaced by 'slices'`
+)
+
+func _(m, a map[string]string) {
+	maps.Clone(m) // want `golang.org/x/exp/maps.Clone\(\) can be replaced by maps.Clone\(\)`
```

Comment:
I think the linter shouldn't omit the message here, but only at the imports level. This is because there is nothing a user can do on this line to fix it. They can only adjust the imports, and the problem here will automatically be resolved.

Reply:
I can do something for `slices` but `maps` this is more complex.